### PR TITLE
FEATURE: Improve in login and register pages the UI

### PR DESCRIPTION
**Pull request by DavMovsesyan2002**

### CODE REVIEW

File: src/pages/login/index.tsx [hunk 1]
```diff
@@ -1,97 +1,86 @@
...
   const [email, setEmail] = useState<string>('')
   const [password, setPassword] = useState<string>('')
 
   const handleLogin = () => {
     dispatch(usersMiddleware.login({ email, password }))
   }
 
+  const handleContinueWithGoogle = () => {
+    dispatch(usersMiddleware.continueWithGoogle())
+  }
```
Comment (Arthur02V) [inline src/pages/login/index.tsx:24]: onGoogleAuth

File: src/manager/authentication/authManager.ts [hunk 2]
```diff
@@ -20,6 +20,9 @@ const authManager = {
   signUp(params: ISignUpParams) {
     return axiosInstance.post<null, IAxiosResponse<null>>(`${baseURL}/v1/auth/register`, params)
   },
+  continueWithGoogle() {
+    return axiosInstance.post<null, IAxiosResponse<null>>(`${baseURL}/v1/auth/google`)
+  },
```
Comment (Arthur02V) [inline src/manager/authentication/authManager.ts:25]: googleSignIn

**Review by Arthur02V — CHANGES_REQUESTED** [review]

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]
We should not send role on register

**Comment by ArthurO2002** [comment]
Let's add some validator package for example yup validator to not write custom error

File: src/pages/login/index.tsx [hunk 3]
```diff
@@ -1,99 +1,119 @@
...
+                  id="remember-me"
+                />
+              )}
+              name="remember"
+            />
+          </div>
+          <Button
+            size="fl"
+            disabled={isSignInLoading}
+            // onClick={handleLogin}
```
Comment (ArthurO2002) [inline src/pages/login/index.tsx:94]: remove comments

File: src/pages/reset-password/index.tsx [hunk 4]
```diff
@@ -16,7 +18,7 @@ export const ResetPassword = () => {
   const [emailVerificationError, setEmailVerificationError] = useState<string>()
 
   const handleEmailSend = () => {
-    const emailVerification: boolean = /\S+@\S+\.\S+/.test(emailValue)
+    const emailVerification: boolean = testEmail(emailValue)
 
     if (!emailVerification || !emailValue.length) {
       setEmailVerificationError('Email not recognized')
```
Comment (ArthurO2002) [inline src/pages/reset-password/index.tsx:24]: This is not needed anymore

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

**Review by rikarsen — CHANGES_REQUESTED** [review]
image
image

Please review UI Dav jan. Many parts need to clean up and you have console errors

File: src/pages/login/index.tsx [hunk 5]
```diff
@@ -1,99 +1,118 @@
...
+          </Button>
+          {error || errorGoogleSignIn ? (
+            <div className="mt-2.5 w-full">
+              <p className="text-red">{error ?? errorGoogleSignIn}</p>
+            </div>
+          ) : null}
+          <div className="mb-44 mt-7 w-full">
+            <Link href="/reset-password">
+              <Button
+                className="ml-2	font-normal	"
```
Comment (ArthurO2002) [inline src/pages/login/index.tsx:106]: Clear later

**Review by ArthurO2002 — DISMISSED** [review]

**Review by rikarsen — CHANGES_REQUESTED** [review]
fix width from 350px to 380px
add google logo to login with google button

File: src/pages/login/index.tsx (hunk 1 above)
Comment (DavMovsesyan2002) [reply src/pages/login/index.tsx]: Done

File: src/manager/authentication/authManager.ts (hunk 2 above)
Comment (DavMovsesyan2002) [reply src/manager/authentication/authManager.ts]: Done

**Pull request merged**